Add Media Source documentation (#357)
livingdocsIO/livingdocs: reference-docs/project-config/media_types.md

| Property Release Id                        | PropertyReleaseID                                            |
| Property Release Status                    | PropertyReleaseStatus                                        |
| Web Statement of Rights                    | WebStatement                                                 |
| Artwork or Object in the Image structure   | ArtworkOrObject                                              |

## Media Sources

Added in: `release-2021-03`

> This feature is only supported for the `mediaImage` mediaType.

Reviewer: ```suggestion
This feature is only supported for mediaTypes of type `mediaImage`.
```

Reviewer 2: I'd always prioritize the description and then on what we can't do. So I'd move that after the next paragraph.

> With [Media Sources](/reference/media-source-example.md) you can integrate asset research/import into Livingdocs. Instead of going onto another platform like Unsplash and then drag + drop images into Livingdocs, you can have Unsplash assets search results embedded into Livingdocs directly.

Reviewer: ```suggestion
With [Media Sources](/reference/media-source-example.md) you can integrate asset research/import into Livingdocs. Instead of having to log in to another platform like Unsplash and then drag + drop images into Livingdocs, you can have Unsplash assets search results available directly in Livingdocs.
```
